ibm/service/eventnotification/resource_ibm_en_destination_ce.go

```diff
@@ -66,14 +68,29 @@ func ResourceIBMEnCodeEngineDestination() *schema.Resource {
 								Schema: map[string]*schema.Schema{
 									"url": {
 										Type:        schema.TypeString,
-										Required:    true,
+										Optional:    true,
 										Description: "URL of code engine project.",
 									},
 									"verb": {
 										Type:        schema.TypeString,
-										Required:    true,
+										Optional:    true,
 										Description: "HTTP method of webhook.",
 									},
+									"type": {
```

Comment:
Adding "Required" breaks the current user when they upgrade to new version
Can we mark it as optional